public/app/features/alerting/unified/components/AlertManagerPicker.tsx

```diff
@@ -9,43 +9,39 @@ import { AlertManagerDataSource, GRAFANA_RULES_SOURCE_NAME } from '../utils/data
 
 interface Props {
   disabled?: boolean;
-  /**
-   * If true, only show alertmanagers that are receiving alerts from Grafana
-   */
-  showOnlyReceivingGrafanaAlerts?: boolean;
 }
 
 function getAlertManagerLabel(alertManager: AlertManagerDataSource) {
   return alertManager.name === GRAFANA_RULES_SOURCE_NAME ? 'Grafana' : alertManager.name.slice(0, 37);
 }
 
-export const AlertManagerPicker = ({ disabled = false, showOnlyReceivingGrafanaAlerts }: Props) => {
+export const AlertManagerPicker = ({ disabled = false }: Props) => {
   const styles = useStyles2(getStyles);
   const { selectedAlertmanager, availableAlertManagers, setSelectedAlertmanager } = useAlertmanager();
 
   const options: Array<SelectableValue<string>> = useMemo(() => {
-    return availableAlertManagers
-      .filter(({ name, handleGrafanaManagedAlerts }) => {
-        const isReceivingGrafanaAlerts = name === GRAFANA_RULES_SOURCE_NAME || handleGrafanaManagedAlerts;
-        return showOnlyReceivingGrafanaAlerts ? isReceivingGrafanaAlerts : true;
-      })
```

Comment:
We don't need to filter these in the end as we aren't showing the alert manager picker in the end